Enterprise-CMCS/macpro-mdct-seds
# Remove unneeded JSON file

### Description
This PR deletes `auth_job_codes.json`. It contained misleading data, which was never used anywhere. Now it won't confuse anyone in the future.

This JSON file lived next to a bunch of others, which are used to seed the database. However, those files are referenced from serverless.yml. This file isn't, and never was. Additionally, I spun up the app and used Dynamo Local Admin to confirm that no table contained this data.

The data we actually use to map job codes to roles is hardcoded, in `initialLoadFunctions.js`.

### Related ticket(s)
n/a

---
### How to test
1. Log in as an admin user
2. Log in as a regular user
3. Note that the login functions have not been broken

### Important updates


---
### Author checklist
